src/components/team/remove-member-confirm-dialog.tsx

```diff
@@ -0,0 +1,88 @@
+import React, {FunctionComponent} from 'react'
```

Comment:
This is the 'Remove Team Member' confirm dialog. It shows the name/email of the member you are about to remove from your team and asks you to confirm you want to do it.

You can 'Cancel', 'x', or hit 'Escape'. Or you can hit 'Confirm' which will trigger a side-effect action in the machine.